### ISSUE TITLE
As a developer, I would like to have "current year" data points in the API

### ISSUE THREAD

**Issue by saintsoup52** [18F/college-choice#1387]
I mentioned this a couple of months back but I can't find it on issues... 

I think is probably the most important output for me as a developer of tools that will consume the data to be able to have data points that will always be the most current. I will want to most recent graduation rate, netprice, default rate, average salary and median borrowing rates.

Right now each data point is contained in different years and that make extracting the data much more complex.

**Comment by sudharsanmit** [comment]
@saintsoup52 I agree. Atleast, the metadata should have a parameter - current_year so that all queries can be prefixed with that year. Also, when will the data be updated for the years 2015, 2016?
